### DISCUSSION TITLE
一个事业、一个目标、一个信念；创新传播规律；我有一个梦想。

### DISCUSSION THREAD

**Discussion by leslie158** [Benefitspace/dream#5] [Ideas]
一个事业、一个目标、一个信念；创新传播规律；我有一个梦想。人们追随信念，是在追随这个同样属于他们自己的信念——“我有一个梦想”。
DreamCoin是让每一个人获得梦想，实现梦想，我们需要将这个梦想用最友好完整和无私的方式呈现出来，让它成为每一个志愿者和社区用户的梦想。
不用太多，让我们每个人都拥有10亿的Dream。
DreamCoin Contract:
0x3bfE9F4e25ed2793610BD78aFe766871F62AA11c
---------------------------------------------------
One career, one goal, one belief; the law of innovation and communication; I have a dream. People who follow beliefs are following this belief that also belongs to them-"I have a dream."
DreamCoin is to let everyone achieve their dreams and realize their dreams. We need to present this dream in the most friendly, complete and selfless way, and make it the dream of every volunteer and community user.
Not too much, let each of us have 1 billion Dream.
DreamCoin Contract: 0x3bfE9F4e25ed2793610BD78aFe766871F62AA11c

**Comment by leslie158** [comment]
已经持有10万亿Dream：0x06cbda490e96c746234e225f3ed9b06c0e3e14f9
@Benefitspace  要给我空投更多的更多更多的梦想么，哈哈哈

**Reply by westlife29BTC** [reply]
![image](https://user-images.githubusercontent.com/80855175/118388225-bd56f480-b655-11eb-91cf-b1aea538a8e6.png)

**Comment by darktuJJ** [comment]
我认为你应该把电报群和discord和微信群搞起来。。

**Comment by yw-hf** [comment]
你好  我持有五万亿梦想，你可以把空投发给我 我去帮你推空投   微信号q793492678

**Comment by leslie158** [comment]
@Benefitspace 自述文件里面的有个错误 及时修改一下
什么是dream
dreamcoin牧羊犬是一个由社区推动且公平启动的代币
牧羊犬这里应该是梦想币

**Comment by westlife29BTC** [comment]
0xeaf5deff30ccc822c36eb65436481d10806918e9
这个地址10%的币是项目方自己的还是之前买的？

**Reply by leslie158** [reply]
0xf6c07f61974c59f675f209c8f44e3d2f813460fdde2da6bc680f7b1682080ee7
查了哈希发现这个地址捐赠了10个BNB

**Reply by westlife29BTC** [reply]
原来如此，对了，你有没有discord或者电报，搞个推特啥的，好推广啊

**Reply by leslie158** [reply]
搞起来
